realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -230,6 +232,25 @@ private static synchronized void removeSession(SyncConfiguration syncConfigurati
         }
     }
 
+    /**
+     * Retruns the all valid sessions belonging to the user.
```

Comment:
`SynUser::all_sessions` are the sessions maintained by Object Store, but here the `sessions` is maintained by the java layer. So what is our plan? Are we going to remove the `SyncManager.sessions` to use the `SyncUser::all_sessions` instead?